homeassistant/loader.py

```diff
@@ -1077,43 +1128,49 @@ def _get_platform_cached(self, full_name: str) -> ModuleType | None:
 
     def get_platform(self, platform_name: str) -> ModuleType:
         """Return a platform for an integration."""
-        if platform := self._get_platform_cached(f"{self.domain}.{platform_name}"):
+        if platform := self._get_platform_cached_or_raise(
+            f"{self.domain}.{platform_name}"
+        ):
             return platform
         return self._load_platform(platform_name)
 
-    def platform_exists(self, platform_name: str) -> bool | None:
-        """Check if a platform exists for an integration.
+    def platform_missing(self, platform_name: str) -> bool:
```

Comment:
this function isn't used anymore?

Reply:
I thought I deleted it but I think I screwed up the merge

Reply:
Yeah I deleted it in the other branch but not here

Reply:
e32b2d70f6f071ccec99c4c1f7b11d5cfda9a6a3